src/profiler/profiler.cpp

```diff
@@ -279,7 +279,7 @@ bool Profiler::sampleTarget(SAMPLE_TYPE timeSpent, SymbolInfo *syminfo)
 			stack.addr[stack.depth++] = ip;
 		first = false;
 
-		BOOL result = dbgHelp->StackWalk64(
+		BOOL result2 = dbgHelp->StackWalk64(

```

Comment:
It would be better to put the previous variable in a narrower scope instead.

Reply:
How about if I rename them both so that neither of them has a generic name?

Reply:
Would you rather have arbitrary code blocks to remove the variable from the function-level scope? I'm not going to fight you about what you want here, I just don't see an option that is clearly better in all respects.

Reply:
I do, and it does make sense to me to reduce visibility of declarations as much as possible, even within functions, but perhaps that's an unusual preference. Someone more pedantic about code cleanliness would probably suggest that if we are at a point where variable names clash within a function, it's time to refactor that function into several.

In any case this is such a minor issue that I'm happy with any solution. I would ask though that the fix is in its own commit (or grouped with other fixes with the same purpose) and the commit message documents why the change is done.